contracts/chain-adapters/Arbitrum_CustomGasToken_Adapter.sol

```diff
@@ -337,12 +344,11 @@ contract Arbitrum_CustomGasToken_Adapter is AdapterInterface, CircleCCTPAdapter
     }
 
     function _from18ToNativeDecimals(uint256 amount) internal view returns (uint256) {
-        uint8 nativeTokenDecimals = L1_INBOX.bridge().nativeTokenDecimals();
```

Comment:
Can you remove `nativeTokenDecimals` from the interface now

Reply:
https://github.com/across-protocol/contracts/pull/699/commits/a6f7e3fc0c9e33a5c296127211c38a3edc32b266